### ISSUE TITLE
Event notification time form field too small

### ISSUE THREAD

**Issue by miaulalala** [nextcloud/calendar#4870]
### Steps to reproduce

I cannot see the time I have set as the in-/decrease buttons overlay the numbers:

![image](https://user-images.githubusercontent.com/7427347/210213293-dc3217fe-94a6-4584-b4d5-93f074fa50d9.png)

### Expected behavior

I can see the time, use the form field or the buttons to update the notification time

### Actual behaviour

I can't see what I'm typing

### Calendar app version

v4.2.0

### CalDAV-clients used

Web

### Browser

Chrome

### Client operating system

Ubuntu

### Server operating system

Ubuntu

### Web server

Apache

### Database engine version

MySQL

### PHP engine version

PHP 8.0

### Updated from an older installed version or fresh install

Updated from an older version

**Comment by aledef** [comment]
+1
